apps/code-examples/src/app/app.component.html

```diff
@@ -13,12 +13,58 @@
       <li><a routerLink="forms/checkbox/basic">Checkbox (basic)</a></li>
     </ul>
   </li>
+
   <li>
     Layout
     <ul>
       <li><a routerLink="layout/box/basic">Box (basic)</a></li>
     </ul>
   </li>
+
+  <li>
+    Lookup
+    <ul>
+      <li>
+        Autocomplete
+        <ul>
+          <li><a routerLink="lookup/autocomplete/basic">Basic</a></li>
```

Comment:
Probably doesn't matter, but should these items be in alphabetic order like the parent list? Or does the logic of basic, then advanced, then custom trump alphabetic order?

Reply:
This is going to be updated to use a different UI layout in the future, so it doesn't really matter for now.